server/src/main/kotlin/org/javacs/kt/SourcePath.kt

```diff
@@ -13,6 +13,7 @@ import org.jetbrains.kotlin.psi.KtFile
 import org.jetbrains.kotlin.resolve.BindingContext
 import org.jetbrains.kotlin.resolve.CompositeBindingContext
 import org.jetbrains.kotlin.resolve.scopes.DescriptorKindFilter
+import java.lang.Exception
```

Comment:
`java.lang.*` classes are automatically imported, so I think this import is redundant.

Reply:
Not really sure how this showed up here. IntelliJ shouldn't auto import this sort of stuff. But then again, I wouldn't add this myself either 🤷 Anyway, it's fixed now.